config/jobs/kubernetes/kops/kops-periodics-misc2.yaml

```diff
@@ -1204,8 +1204,6 @@ periodics:
       args:
       - ./tests/e2e/scenarios/aws-ebs-csi/run-test.sh
```

Comment:
For the scenario script tests we either need to keep the KOPS_IRSA env var or pass an OVERRIDES env var containing the same --override flags. As-is, the `extra_flags` argument to `build_test()` and `presubmit_test()` isn't used when a scenario script is specified.